.github/workflows/comments-for-fun.yml

```diff
@@ -1,6 +1,9 @@
 name: SomeFunnyWorkflow
 
 'on':
+  pull_request_review:
+  issue_comment:
+  pull_request_review_comment:
```

Comment:
So, `issue_comment` is safe. But both `pull_request_review` and `pull_request_review_comment` could be launched, if the owner do the things.

It doesn't expose the secrets, but I believe, that they must not be launched at all from the forks.